### PR TITLE
Adding Infinispan documentation

### LINKED ISSUE #170: Add documentation for HotRod
### Description

With #169 in place, there is now missing some documentation on how to use it in load tests. 

This should answer the following information: 

* How to configure the setup to use Infinispan/HotRot
* Provide information for the "mental model" how the wiring are
* Information on how to debug problems / use the setup for developing new test

In addition, the previous store CockroachDB is in the ominous [other.adoc](https://github.com/keycloak/keycloak-benchmark/blob/main/doc/kubernetes/modules/ROOT/pages/other.adoc#accessing-the-cockroachdb-inside-minikube) file, so we need come up with a new structure that is better suited for users and makes use of the new possibilities that Antora provides.

### Discussion

_No response_

### Motivation

It should be ready-to-use for those who want to run load tests.

### Details

_No response_

### PR DESCRIPTION
**Pull request by ahus1**
This also tries to make best use of the Antora features to write small and self-contained pages that answer to a single purpose/user's need.

Closes #170

### CODE REVIEW

**Comment by ahus1** [comment]
@mhajas - I created a PR that contains some new structures that are helpful for Antora, as well as adding the things I know about the Infinispan setup.

To edit the content, I recommend IntelliJ and the AsciiDoc plugin. The preview should be ok to edit content. 

To create a local preview, follow [the readme in the `antora` folder](https://github.com/keycloak/keycloak-benchmark/tree/main/antora).

Thinking a bit more about Infinispan and how it advertises its address, I assume that the setup with the node port doesn't work. Please give it a test run. If it doesn't work, I could remove the contents from the docs plus also the node part. 

For any changes to my suggested content, I suggest you create a PR to this PR. Please let me know what you think.

**Review by mhajas — CHANGES_REQUESTED** [review]
Thanks for the Pr @ahus1! The doc looks good to me, I just added a few comments with typos/nitpicks. 

Regarding node port. I was able to connect to the ISPN server using `infinispan-cli` so I would say it works currently. 

Regarding advertising the different IP address warnings, I created an issue for [upgrading ISPN version](https://github.com/keycloak/keycloak-benchmark/issues/172) to the newer one. As part of this issue, we will also check whether the problem is still present and if yes we will fix it (In the testsuite the issue was not present anymore with the newer version but let's see).

File: doc/kubernetes/modules/ROOT/pages/storage/infinispan.adoc [hunk 1]
```diff
@@ -0,0 +1,54 @@
+= Using Infinispan storage
+:description: The deployment of Keycloak can use an external infinispan instance when deployed with the new map storage.
+
+{description}
+
+This is using the new map storage.
```
[the remark this answers is not in the document]
Comment (ahus1) [reply doc/kubernetes/modules/ROOT/pages/storage/infinispan.adoc:6]: yes, removing it in the next commit.

**Comment by ahus1** [comment]
@mhajas - thanks for your comment; accepted your changes. Also added the the node port can be used from the CLI.

**Review by mhajas — APPROVED** [review]
Thanks @ahus1

**Pull request merged**